src/Mod/Path/PathScripts/PathWaterline.py

```diff
@@ -25,27 +25,37 @@
 
 from __future__ import print_function
 
+__title__ = "Path Waterline Operation"
+__author__ = "russ4262 (Russell Johnson), sliptonic (Brad Collette)"
+__url__ = "http://www.freecadweb.org"
+__doc__ = "Class and implementation of Waterline operation."
+__contributors__ = ""
+
 import FreeCAD
+from PySide import QtCore
+
+# OCL must be installed
+try:
+    import ocl
+except ImportError:
+    msg = QtCore.QCoreApplication.translate("PathWaterline", "This operation requires OpenCamLib to be installed.")
+    FreeCAD.Console.PrintError(msg + "\n")
+    raise
```

Comment:
I think you are supposed to raise the same type of exception, so `raise ImportError`.

Reply:
Fixed.  Also Fixed on PR #3327 .